config.json

```diff
@@ -186,6 +186,18 @@
       "values": [ "debug", "release", "checked" ],
       "defaultValue": "debug"
     },
+    "EnableProfileGuidedOptimization": {
+      "description": "Enables IBC profile optimizations if profile data are available.",
+      "valueType": "property",
+      "values": [ true, false ],
+      "defaultValue": false
+    },
+    "OptimizationDataDir": {
+      "description": "Sets the path where the build should look for IBC profile data.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
```

Comment:
Thanks - I really like the idea of inheriting shared properties such as these from a common source. From the issue though it looks like this support isn't there yet? I'd like to move forward with the current implementation for now. We should be able to remove these entries from the local config.json later once they're implemented in a shared config, right?